test/integration/test_phone_number_lookup.py

```diff
@@ -0,0 +1,168 @@
+                os.environ['BW_NUMBER'],
+                os.environ['VZW_NUMBER']
+                # os.environ['BW_INVALID_TN_LOOKUP_NUMBER']
+            ],
+        ) 
+
+        create_lookup_response: CreateLookupResponse = self.api_instance.create_lookup(self.account_id, lookup_request, _return_http_data_only=False)
+        self.assertEqual(create_lookup_response[1], 202)
+        self.assertIs(type(create_lookup_response[0].status), LookupStatusEnum)
+        self.assertEqual(create_lookup_response[0].status, LookupStatusEnum("IN_PROGRESS"))
+        self.assertIs(type(create_lookup_response[0].request_id), str)
+        self.assertRegex(create_lookup_response[0].request_id, r'^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$')
+        
+        get_lookup_status_response: LookupStatus = self.api_instance.get_lookup_status(self.account_id, create_lookup_response[0].request_id, _return_http_data_only=False)
+        self.assertEqual(get_lookup_status_response[1], 200)
+
+        # make the request again without _return_http_data_only=False to just get the LookupStatus model
+        get_lookup_status_response: LookupStatus = self.api_instance.get_lookup_status(self.account_id, create_lookup_response[0].request_id)
+        get_lookup_status_response_attempts = 1
+        while get_lookup_status_response.status != LookupStatusEnum('COMPLETE'): 
+            # Raise an error if it takes more than 5 requests to get COMPLETE status
+            if get_lookup_status_response_attempts == 5:
+                raise Exception(f'Took too long to get phone number lookup \'COMPLETE\' status. Aborting test after {get_lookup_status_response_attempts} attempts.')
+            time.sleep(2)
+
+            get_lookup_status_response: LookupStatus = self.api_instance.get_lookup_status(self.account_id, create_lookup_response[0].request_id)
+            get_lookup_status_response_attempts += 1
+
+        self.assertEqual(get_lookup_status_response.request_id, create_lookup_response[0].request_id)
+        self.assertIs(type(get_lookup_status_response), LookupStatus)
+        
+        # Assert that each result is of type LookupResult
+        for i in range(len(get_lookup_status_response.result)):
+            self.assertIs(type(get_lookup_status_response.result[i]), LookupResult)
+
+        # Check the information for a Bandwidth TN
+        bw_lookup_result: LookupResult = get_lookup_status_response.result[0]
+        self.assertEqual(bw_lookup_result.response_code, 0)
+        self.assertIs(type(bw_lookup_result.message), str)
+        self.assertEqual(bw_lookup_result.e_164_format, os.environ['BW_NUMBER'])
+        self.assertIs(type(bw_lookup_result.formatted), str)
+        self.assertTrue(bw_lookup_result.country == "US" or bw_lookup_result.country == "Canada")
+        self.assertTrue(bw_lookup_result.line_type == "Mobile" or bw_lookup_result.line_type == "Fixed")
+        self.assertIn("Bandwidth", bw_lookup_result.line_provider)
```

Comment:
Following [Don't Repeat Yourself (DRY)](https://deviq.com/principles/dont-repeat-yourself) principle the above 7 assertions and similar ones below can be abstracted to a single function removing duplication.

Reply:
Good call out - added a reusable `validateResult` function

Reply:
Clarification on `test logic and country/line type` - In instances where you expect it to be `US` it might be `Canada` and the check will still pass. Similarly when you expect `line_type` to be `Mobile` actual it might be `Fixed` and check will still pass. The larger point is our `asserts` should have minimal logic to avoid introducing unintentional defects in test code.